RxExample/RxExample/Examples/WikipediaImageSearch/Views/WikipediaSearchViewController.swift

```diff
@@ -13,15 +13,10 @@ import RxCocoa
 #endif
 
 class WikipediaSearchViewController: ViewController {
+    private let searchController = UISearchController(searchResultsController: UITableViewController())
+    
+    @IBOutlet weak var searchBarContainer: UIView!
     
-    private var resultsTableView: UITableView {
```

Comment:
Is there some reason why we are removing `resultsTableView` and `searchBar ` properties instead of just saying 
```
private var resultsViewController: UITableViewController {
     return (self.searchController.searchResultsController as?  UITableViewController)!
}
private var resultsTableView: UITableView {
        return resultsViewController.tableView!
}
private var searchBar: UISearchBar {
       return self.searchController.searchBar
}
```

I don't like the `!`, but it seems that we are doing that in the code anyway.